### PR TITLE
Update multiple sections

### PR DESCRIPTION
**Pull request by adsantos**
Updated spec_n_days, spec_user and spec_project. The last one is just replacing `p` with `#`.

### CODE REVIEW

File: spec_n_days.json [hunk 1]
```diff
@@ -21,32 +17,9 @@
...
-                        },
-                        {
-                            "name": "Sunday",
-                            "type": "Daily",
-                            "priority": null,
-                            "date": "2015-07-12",
-                            "project": null,
-                            "items": [2,3],
+                            "items": [1,2,3],
                             "isReorderable": true,
```
Comment (anupcowkur) [inline spec_n_days.json]: shouldn't `isReorderable` and `labels` fields be removed as well?

File: spec_project.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
                     "freezeDate" : null,
```
Comment (anupcowkur) [inline spec_project.json]: the `behaviour` field in this file still uses `p` instead of `#` and needs to be updated.

File: spec_n_days.json (hunk 1 above)
Comment (adsantos) [inline spec_n_days.json]: Yes, I missed them on this test, thanks.

File: spec_user.json [hunk 3]
```diff
@@ -6,7 +6,10 @@
                 "The supported user query is 'u:{user}'",
```
Comment (anupcowkur) [inline spec_user.json]: In this file, tests `1`,`2` and `3` also need to be updated to use the new section format.

File: spec_project.json [hunk 4]
```diff
@@ -3,8 +3,8 @@
         {
             "name": "project",
             "behaviour": [
-                "The supported project query is: 'p:{project-name}'.",
-                "All whitespace outside the {project-name} is ignored, e.g. 'p:work' == 'p : work'.",
+                "The supported project query is: '#:{project-name}'.",
```
Comment (Garee) [inline spec_project.json]: `:` is no longer necessary.

**Comment by adsantos** [comment]
All fixed

**Pull request merged**